vm/src/thread.rs

```diff
@@ -241,6 +241,7 @@ impl RootedThread {
                 stack: Stack::new(),
                 record_map: FieldMap::new(),
                 hook: None,
+                stack_size_limit: VmIndex::max_value(),
```

Comment:
I think it would be implicit for the specific datatype. Doc comments could help though!